cli.mjs

```diff
@@ -395,3 +480,54 @@ const getDataRef = (studyID, participantID) =>
 // Get a reference to a participant's specific experiment data document in Firestore
 const getExperimentRef = (studyID, participantID, experimentID) =>
   getDataRef(studyID, participantID).doc(experimentID);
+
+// Get a reference to a registered study
+const getRegisteredfStudyRef = (studyID) => FIRESTORE.collection(REG_STUDY_COL).doc(studyID);
+
+// Get current registered participant array under the StudyID
+const getRegisteredParticipantArr = (studyID) => {
+  const res = getRegisteredfStudyRef(studyID)
+    .get()
+    .then((data) => {
+      if (data["_fieldsProto"] != undefined) {
+        return data["_fieldsProto"]["registered_participants"]["arrayValue"]["values"]
+          .filter((item) => item.valueType === "stringValue")
+          .map((item) => item.stringValue);
+      } else {
+        return [];
+      }
+    });
+
+  return res;
+};
+
+// Register new participantID under the provided studyID
+const registerNewParticipant = async (studyID, participantID) => {
+  const currParticipants = await getRegisteredParticipantArr(studyID);
+  const newParticipantArray = [...currParticipants, participantID];
+  const newData = { registered_participants: newParticipantArray };
+  getRegisteredfStudyRef(studyID).update(newData);
+  console.log(
+    `Successfully added study and participant. Current participantIDs under study ${studyID}: \n${newParticipantArray.join("\n")}`
+  );
+};
+
+// Add new participantID under studyID to Firestore under registered_studies,
+//  creates new study if studyID doesn't exist
+const addStudyAndParticipant = async (studyID, participantID) => {
```

Comment:
Since we're in an async function anyways, instead of using callbacks can we instead us the `await` syntax throughout instead of `.then`?

Reply:
This makes sense! I will refactor this.